resources/helm-broker-repo/development/sync-bundles.sh

```diff
@@ -2,10 +2,12 @@
 
 NAMESPACE=kyma-system
 
-PROVISIONING_DIR="$(dirname $0)/../provisioning"
+BASE_DIR="$(dirname $0)/.."
```

Comment:
IMO it should not be called a base dir because u go outside of this directory. What's more, this will result in a relative path (https://www.electrictoolbox.com/bash-script-directory/)
because of that in our scripts, we are using the absolute path to make sure that it will always work, see https://github.com/kyma-project/kyma/blob/master/installation/scripts/installer.sh#L5. Please follow this pattern, so it could look like this one:

```bash
#!/usr/bin/env bash

NAMESPACE=kyma-system

readonly CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
readonly PROVISIONING_DIR="${CURRENT_DIR}/../provisioning"
readonly BUNDLES_DIR="${CURRENT_DIR}/../bundles"

bash ${PROVISIONING_DIR}/provision-bundles.sh ${BUNDLES_DIR}

HELM_BROKER_POD_NAME=$(kubectl get po -n ${NAMESPACE} -l app=core-helm-broker -o jsonpath="{.items[0].metadata.name}")

kubectl delete pod ${HELM_BROKER_POD_NAME} -n ${NAMESPACE}

```